### PR TITLE
Add docker compose to easily allow viewing changes locally

### PR DESCRIPTION
**Pull request by Herkarl**
## Describe your changes

Currently we have the problem that it is fairly hard to view changes you make to the content locally. This fixes this by adding a docker compose file that sets up all necessary systems locally and links them in the correct way.

## Checklist before requesting a review

- [x] I have performed a self-review of my changes
- [ ] I have updated both Swedish and English pages (if not motivate why)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,19 @@ Klona, gaffla (forka) repot (eller redigera i webbläsaren), gör en ny branch o
...
 
+## Köra lokalt 
+
+Repot har en docker-compose fil som sköter all setup med `bawang` och `taitan`, så om du vill redigera hemsidan lokalt och vill se dina ändringar innan du pushar så är det bara att köra
+```bash
+docker compose up --build
+```
+Därefter så kommer sidan vara tillgänglig på `localhost:8000`. Om du ändrar på en sida så är det bara att relodea för att se dina ändringar.
+
+För tillfället (24-02-15) tar det ett litet tag att bygga första gången du kör det, då `bawang` använder en gammal version, men det löses förhoppningsvis snart.
```
Comment (mathiasmagnusson) [inline README.md:32]:
```suggestion
För tillfället (24-02-15) tar det ett litet tag att bygga första gången du kör det, då `bawang` använder en gammal version, men det löses förhoppningsvis snart™.
```

File: README.md [hunk 2]
```diff
@@ -21,6 +21,19 @@ Klona, gaffla (forka) repot (eller redigera i webbläsaren), gör en ny branch o
 
 Vi har stängt av commits direkt till master.
 
+## Köra lokalt 
+
+Repot har en docker-compose fil som sköter all setup med `bawang` och `taitan`, så om du vill redigera hemsidan lokalt och vill se dina ändringar innan du pushar så är det bara att köra
+```bash
+docker compose up --build
+```
+Därefter så kommer sidan vara tillgänglig på `localhost:8000`. Om du ändrar på en sida så är det bara att relodea för att se dina ändringar.
```
Comment (mathiasmagnusson) [inline README.md:30]:
```suggestion
Därefter så kommer sidan vara tillgänglig på `localhost:8000`. Om du ändrar på en sida så är det bara att ladda om webbläsaren för att se dina ändringar.
```

File: README.md (hunk 1 above)
Comment (Herkarl) [reply README.md:32]: good change

**Review by mathiasmagnusson — APPROVED** [review]
No trailing newline but I'll let it slide this time. 👽

**Pull request merged**